backend/app/views/spree/admin/adjustment_reasons/index.html.erb

```diff
@@ -33,7 +33,7 @@
             <%= adjustment_reason.name %>
           </td>
           <td class="align-center">
-            <%= Spree.t(adjustment_reason.active? ? :active : :inactive) %>
+            <%= Spree::AdjustmentReason.human_attribute_name(adjustment_reason.active? ? :active : :inactive) %>
```

Comment:
I was iffy about this.  active is an attribute on the model (a boolean) but inactive is not.  I did add it in though to make this work obviously.